examples/optimize-qcbm-circuit.yaml

```diff
@@ -80,8 +80,10 @@ spec:
           - n-qubits: "{{workflow.parameters.n-qubits}}"
           - n-layers: "{{workflow.parameters.n-layers}}"
           - topology: "{{workflow.parameters.topology}}"
-          - epsilon: "0.000001"
           - distance-measure-specs: "{'module_name': 'zquantum.core.bitstring_distribution', 'function_name': 'compute_clipped_negative_log_likelihood'}"
+          # - distance-measure-specs: "{'module_name': 'zquantum.core.bitstring_distribution', 'function_name': 'compute_mmd'}"
+          - distance-measure-parameters: "{'epsilon': 0.000001}"
+          # - distance-measure-parameters: "{'sigma': 1}"
```

Comment:
I don't think this commented code should stay here :)
Also, perhaps it's better to write `0.000001 ` as `1e-6`to make it more readable?

Reply:
Right, [done](https://github.com/zapatacomputing/z-quantum-qcbm/pull/14/commits/695aee6489b83728394684f91895c9b7508b17e5) it! 
Also, there's another line commented below (`optimizer-specs`), I didn't delete it since it was already there, do you know if it's needed?

Reply:
It's not needed, but I would leave it since this is for demo purposes.